### PR TITLE
Fixed some lang typos

### PR DESCRIPTION
**Pull request by Bitti09**
This change is [Reviewable](https://reviewable.io/reviews/ryuzakikk/notecrypt/18)

### CODE REVIEW

File: app/src/main/res/values-de-rDE/strings.xml [hunk 1]
```diff
@@ -82,10 +82,10 @@
     <string name="action_search">Suche</string>
     <string name="action_starred">Wichtig</string>
     <string name="action_remove">Entfernen</string>
-    <string name="action_overwrite">Überschreibem</string>
-    <string name="overwrite_message">Datei mit dem Namen existiert, wird nun überschrieben</string>
+    <string name="action_overwrite">Überschreiben</string>
+    <string name="overwrite_message">Eine Datei mit dem Namen existiert bereits,sie wird nun überschrieben</string>
```
Comment (RyuzakiKK) [inline app/src/main/res/values-de-rDE/strings.xml]: Here there should be a space after the comma?

**Review by RyuzakiKK — COMMENTED** [review]
Thank you very much!

Can you check the comment I made? I think you should just add a space before "sie".

File: app/src/main/res/values-de-rDE/strings.xml (hunk 1 above)
Comment (Bitti09) [reply app/src/main/res/values-de-rDE/strings.xml]: ops, done now

**Comment by RyuzakiKK** [comment]
Thanks.

**Pull request merged**